developer/src/kmc-kmn/src/compiler/messages.ts

```diff
@@ -47,11 +47,26 @@ export class CompilerMessages {
   static Fatal_UnexpectedException = (o:{e: any}) => m(this.FATAL_UnexpectedException, `Unexpected exception: ${(o.e ?? 'unknown error').toString()}\n\nCall stack:\n${(o.e instanceof Error ? o.e.stack : (new Error()).stack)}`);
   static FATAL_UnexpectedException = SevFatal | 0x1000;
 
-  static Fatal_MissingWasmModule = () => m(this.FATAL_MissingWasmModule, `Could not instanatiate WASM compiler module`);
+  static Fatal_MissingWasmModule = () => m(this.FATAL_MissingWasmModule, `Could not instantiate WASM compiler module or not initted`);
```

Comment:
```suggestion
  static Fatal_MissingWasmModule = () => m(this.FATAL_MissingWasmModule, `Could not instantiate WASM compiler module or initialization failed`);
```

We'll be able to add the exception here also as a parameter.